### PR TITLE
NoUnsetOnPropertyFixer - fix for function call with class constant inside

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: src/Fixer/LanguageConstruct/NoUnsetOnPropertyFixer.php [hunk 1]
```diff
@@ -197,7 +197,22 @@ private function createTokensSubSet(Tokens $tokens, $startIndex, $endIndex)
      */
     private function doTokensInvolveProperty(Tokens $tokens)
     {
-        return $tokens->isAnyTokenKindsFound([T_PAAMAYIM_NEKUDOTAYIM, T_OBJECT_OPERATOR]);
+        for ($index = 0; $index < $tokens->count(); ++$index) {
+            if ('(' === $tokens[$index]->getContent()) {
+                $index = $tokens->findBlockEnd(
+                Tokens::BLOCK_TYPE_PARENTHESIS_BRACE,
```
Comment (SpacePossum) [inline src/Fixer/LanguageConstruct/NoUnsetOnPropertyFixer.php]: indenting is bit off here I think

**Comment by SpacePossum** [comment]
looking good, please fix the CS minor, Travis failure is not related (existing test is to slow)

**Comment by dmvdbrugge** [comment]
Does this fix #3843 too? (Would be awesome 😉)

**Comment by kubawerlos** [comment]
Unfortunately no, I see a good test case there so will try to solve it as well.

**Comment by dmvdbrugge** [comment]
I guess we should mention here that this is being superseeded by #4022? 😉

**Pull request closed without merging**